src/components/timeline/Event.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+interface EventProps {
+  title: string;
+  startTime: string;
+  endTime: string;
+  children: React.ReactNode;
+}
+
+const Event = ({ title, startTime, endTime, children }: EventProps): JSX.Element => {
+  const [expanded, setExpanded] = useState(false);
+
+  const toggleExpanded = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <StyledEvent>
+      <EventHeader onClick={toggleExpanded}>
+        <EventTime>{startTime}</EventTime>
+        <EventIndicator className="indicator" />
```

Comment:
Using a class could be problematic on principle, as it won't be hashed or anything. That way, it technically speaking could be targetted by other styling. Even if that is highly unlikely, there are other (safer) ways we can do this styling